lib/rewrite.js

```diff
@@ -293,15 +288,12 @@ exports.search = function(js, searchRule) {
   // remove if one exists as the first line
   js = helpers.removeShebang(js);
 
-  var pattern = unwrapNode(esprima.parse(searchRule, {
+  var pattern = unwrapNode(parser.parse(searchRule, {
     raw: true
   }));
 
   var matches = [];
-  falafel(js, {
-    raw: true,
-    loc: true
```

Comment:
I'm not sure...all the tests pass.

I can add it back, but it feels weird to do it without a test failing.